hnn_core/neuron.py

```diff
@@ -263,7 +263,7 @@ def _build(self):
         _create_parallel_context()
 
         # load mechanisms needs ParallelContext for get_rank
-        load_custom_mechanisms()
+        # load_custom_mechanisms()
```

Comment:
This is the change I was trying to show

Reply:
thanks, indeed. I have to do this on my laptop for some reason. I can't figure out what's wrong with my neuron install

Reply:
I saw this behavior too. Make sure you don't have multiple x86_64 directories. I had `./x86_64` and `./mod/x86_64`

`load_custom_mechanisms()` expects only `./mod/x86_64`, but NEURON will try loading `./x86_64` on import and then `load_custom_mechanisms()` would fail.

Reply:
interesting, indeed. That fixed it. I wonder how the two directories got created. We can have a more proper fix later if this seems like a recurring issue.